packages/components/src/components/form/bal-radio/bal-radio-group/bal-radio-group.tsx

```diff
@@ -319,6 +357,14 @@ export class RadioGroup implements ComponentInterface, Loggable {
     }
   }
 
+  private onOptionChange = async () => {
+    try {
+      this.setRadioTabindex(this.value)
+    } catch (e) {
+      console.warn('[WARN] - Could not read radio options')
```

Comment:
- [ ] I think we should fail silently or what do you think? if there are none we render none :-)